raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -46,7 +46,7 @@ contract NettingChannelContract {
         (success, balance) = data.deposit(amount);
 
         if (success == true) {
-            ChannelNewBalance(data.token, msg.sender, balance, 0);
+            ChannelNewBalance(data.token, msg.sender, balance, block.number);
```

Comment:
I thought we wanted to get rid of the block number in events(https://github.com/raiden-network/raiden/issues/450)? And in the test below you're not checking for the block number anyway, so why not just remove is from this event?